pkgs/development/libraries/kpmcore/default.nix

```diff
@@ -29,6 +29,10 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     substituteInPlace src/util/CMakeLists.txt \
       --replace \$\{POLKITQT-1_POLICY_FILES_INSTALL_DIR\} $out/share/polkit-1/actions
+    substituteInPlace src/backend/corebackend.cpp \
+      --replace /usr/share/polkit-1/actions/org.kde.kpmcore.externalcommand.policy $out/share/polkit-1/actions/org.kde.kpmcore.externalcommand.policy
+    substituteInPlace src/util/externalcommandhelper.cpp \
+      --replace "(trustedPrefixes.find(prefix.path()) == trustedPrefixes.end())" "(!prefix.path().startsWith(QStringLiteral(\"/nix/store\")) && !prefix.path().startsWith(QStringLiteral(\"/run/current-system/sw\")))"
```

Comment:
I prefer to create a patch file with description for this. It's not quite trivial.

Also isn't allowing everything in `/nix/store` insecure? For Nix, `allowed-users` are not necessary `trusted-users`. So there are some users can build things with nix-daemon but doesn't have root access.
I think we should only allow `/run/current-system` if it works. Privileged programs from desktop environments should be installed system-wide.

Reply:
Unfortunately only allowing `/run/current-system` does not work. Not sure how big of a security concern it is though given that trusted prefixes are a recent addition, and root permissions are still needed to actually modify partitions